app/elements/pw-shell.html

```diff
@@ -250,13 +286,53 @@
 
       // Save the initial URL so that we don't double-render the first load.
       this._currentPageURL = this._sectionRoute.path;
+
+      // Save which version of docs we're looking at, since we need to
+      // maintain this state across the app.
+      this.updateDocsVersion();
+
+      // Update the version when the version selector is used.
+      this.$.versionSelect.addEventListener('iron-select', function(event) {
+        var newSelection = event.detail.item.textContent;
+        if (newSelection === this.activeDocsVersion)
+          return;
+        this.activeDocsVersion = newSelection;
+        window.location = '/' + this.activeDocsVersion + '/start/';
```

Comment:
Oof, so I kind of rely on the whole page to be reloaded so that the mobile nav bar gets re-pulled from the server. I'm not sure if @keanulee has a better idea :(